application/apps/rustcore/rs-bindings/Cargo.toml

```diff
@@ -28,11 +28,12 @@ node-bindgen = {git = "https://github.com/infinyon/node-bindgen.git", branch = "
 # node-bindgen = { version = "5.0", features = ["serde-json"] }
 processor = { path = "../../indexer/processor" }
 serde = { version = "1.0", features = ["derive"] }
-serde_json = "1.0"
+serde_json = "^1.0"
```

Comment:
what does this? this is usually not used in cargo.toml files

Reply:
```
Caret requirements are an alternative syntax for the default strategy, ^1.2.3 is exactly equivalent to 1.2.3.
```
That's equal to `1.0` as far as I understand